action_list_sample/header.js

```diff
@@ -9,33 +9,80 @@ import {
 export default class Header extends Component {
     render() {
         return (
-            <View style={ style.container }>
+            <View style={ style.toolbar }>
                 <Image
-                  style={style.image}
-                  source={require('./img/trident-large.png')}
-                />
-                <Text style={ style.text }>Welcome to Action List</Text>
+                  style={style.toolbarTrident}
+                  source={require('./img/trident-large.png')}/>
+
+                <Text style={ style.toolbarTitle}>IU Bloomington</Text>
+
+                  <Image
+                    style={style.toolbarSearch}
+                    source={require('./img/search.png')}/>
+
+                <Image
+                  style={style.toolbarButton}
+                  source={require('./img/hamburger.png')}/>
             </View>
         )
     }
 }
 
+// const style = StyleSheet.create({
```

Comment:
If this stylesheet isn't needed, remove it (it will remain in the commit history if you need to get it back).